apps/app/src/app/account/account-nua/account-nua.component.ts

```diff
@@ -56,8 +59,11 @@ export class AccountNUAComponent {
   }
 
   goToActivity(client: NUAClient): void {
-    this.stateService.account
+    this.isNuaActivityEnabled
```

Comment:
a bit overkill (the button is invisible if the feature is not enabled), but no big deal

Reply:
I know, it's a security habit I have from Onna 😅 
And actually, I didn't go through it all as I didn't protect the ability to route to nua activity page with a guard, but I felt like that was good enough 😉